src/shared/utils/handlers/HandlerProblemResponseStatusCode.ts

```diff
@@ -0,0 +1,21 @@
+import { AxiosResponse } from "axios";
+
+function validateStatus(status: number) {
+  return (status >= 200 && status < 300) || status === 302 || status === 404;
+}
+
+function handleResponseStatus(response: AxiosResponse) {
+  if (response.status === 302) {
+    return response.data;
+  } else {
+    if (response.status === 404) {
+      return []
+    } else {
+      alert(
+        'Não foi possível carregar os tipos de problemas. Tente novamente!'
+      );
+    }
+  }
+}
+
+export { validateStatus, handleResponseStatus }
```

Comment:
Já temos um handler para isso, por quê foi criado esse novo?  

``` src/shared/utils/handlers/HandlerResponseStatusCodeFound.ts ```

[HandlerResponseStatusCodeFound.ts](https://github.com/The-Bugger-Ducks/help-duck-web/blob/feat/possible-solutions/src/shared/utils/handlers/HandlerResponseStatusCodeFound.ts)

Reply:
por conta da frase do alerta :clown_face: